src/app/tl-mega-menu/tl-mega-menu.component.scss

```diff
@@ -2,6 +2,23 @@
 @import "../shared/styles/tl-variables.scss";
 
 :host {
+  .tl-mega-menu-custom-top {
+    width: calc(var(--wvr-dropdown-menu-width) + 6px);
```

Comment:
Lets try and set the drop down menu padding to 0, and to add padding to the .mega-menu-sections div.